### ISSUE TITLE
Endless loop of GET and HEAD requests on playback

### ISSUE THREAD

**Issue by timoteoponce** [signageos/smil-player#9]
Hello there.

We've ran into the following scenario:
- Install the player in a device
- Enter a SMIL-URL for playback and submit the form
- Player goes into a black screen without any content nor update
- When checking the SMIL-URL hosting, we see an endless loop (every second) of GET and HEAD requests over the SMIL-URL
- The only way to stop the endless requests is to shutdown the device

Workaround:
```
- Switch to a different SMIL-URL
- Make it play
- Switch back to the original SMIL-URL
```
We are guessing that this is an issue with cache, is this perhaps something you have seen before?

Greetings.

**Comment by PetrDusek89** [comment]
Hello,

no we never experienced this issue. Does this behaviour occur with specific smil file or it is random? If its file specific please include it in your response.

**Comment by timoteoponce** [comment]
We have reproduced it with the following steps

- Empty device RPi4 
- Empty player, first boot
- Enter the URL http://deploy.screenfood.com/lightplayer/smil/kristest/kris 
- Endless loop begins

**Comment by timoteoponce** [comment]
Found the problem, but it's a strange issue since in some devices plays well, while in others gets stuck.

The solution is to use full URI paths in all media, originally we had:

```
<seq end="__prefetchEnd.endEvent">
    <seq repeatCount="indefinite">
      <img src="/sf/images/smil_loading_bg.jpg"/>
    </seq>
  </seq>
``` 
and the version that works is 

```
<seq end="__prefetchEnd.endEvent">
    <seq repeatCount="indefinite">
      <img src="https://deploy.screenfood.com/sf/images/smil_loading_bg.jpg"/>
    </seq>
  </seq>
```

**Comment by PetrDusek89** [comment]
Ok thank you for your investigation, i will look into the issue.

**Comment by PetrDusek89** [comment]
Hello, this issue should be resolved in 1.6.x version of the player.
